roles/test_operator/defaults/main.yml

```diff
@@ -0,0 +1,57 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+# defaults file for test_operator
+# All variables within this role should have a prefix of "cifmw_test_operator"
+
+cifmw_test_operator_artifacts_basedir: "{{ cifmw_basedir | default(ansible_user_dir ~ '/ci-framework-data') }}/tests/test_operator"
+cifmw_test_operator_namespace: openstack
+cifmw_test_operator_index: quay.io/openstack-k8s-operators/test-operator-index:latest
+cifmw_test_operator_tempest_registry: quay.io
+cifmw_test_operator_tempest_namespace: podified-antelope-centos9
+cifmw_test_operator_tempest_container: openstack-tempest
+cifmw_test_operator_tempest_image: "{{ cifmw_test_operator_tempest_registry }}/{{ cifmw_test_operator_tempest_namespace }}/{{ cifmw_test_operator_tempest_container }}"
+cifmw_test_operator_tempest_image_tag: current-podified
+cifmw_test_operator_timeout: 60
+cifmw_test_operator_logs_image: quay.io/quay/busybox
+cifmw_test_operator_concurrency: 8
+cifmw_test_operator_cleanup: false
+cifmw_test_operator_dry_run: false
+
+# Please refer to https://openstack-k8s-operators.github.io/test-operator/guide.html#executing-tempest-tests
+cifmw_test_operator_config:
```

Comment:
>  Is expected that other tools will be added to test-operator?

Yes, it is expected. We are in the middle of adding tobiko to the test-operator. 

> If yes, would be good to rename these vars to include _tempest_ to avoid renaming in the future?

Thank you, good point. I updated the code.